source/modules/MEI.stringtab.xml

```diff
@@ -104,9 +181,92 @@
     </attList>
     <remarks xml:lang="en">
       <p>The <att>startid</att> and <att>endid</att> attributes are used to indicate the <gi
-        scheme="MEI">chordMember</gi> elements on which the barre starts and finishes respectively.
-        The fret at which the barre should be created is recorded by the <att>fret</att>
+          scheme="MEI">chordMember</gi> elements on which the barre starts and finishes
+        respectively. The fret at which the barre should be created is recorded by the <att>fret</att>
         attribute.</p>
     </remarks>
   </elementSpec>
-</specGrp>
+  <elementSpec ident="course" module="MEI.stringtab" mode="add">
+    <gloss>individual course tuning information</gloss>
+    <desc xml:lang="en">Describes the tuning of a course on a stringed instrument (e.g., guitar, lute).</desc>
+    <classes>
+      <memberOf key="att.course.log"/>
+    </classes>
+    <content>
+      <rng:zeroOrMore>
+        <rng:ref name="string" />
+      </rng:zeroOrMore>
+    </content>
+  </elementSpec>
+  <elementSpec ident="tuning" module="MEI.stringtab" mode="add">
+    <gloss>instrument tuning information</gloss>
+    <desc xml:lang="en">Describes the tuning of an instrument.</desc>
+    <classes>
+      <memberOf key="att.tuning.log"/>
+    </classes>
+    <content>
+      <rng:zeroOrMore>
+        <rng:ref name="course" />
+      </rng:zeroOrMore>
+    </content>
+  </elementSpec>
+  <elementSpec ident="tabDurSym" module="MEI.stringtab">
+    <desc xml:lang="en">A visual indication of the duration of a <gi scheme="MEI">tabGrp</gi>.</desc>
+    <classes>
+      <memberOf key="att.common" />
+      <memberOf key="att.facsimile" />
+      <memberOf key="att.stringtab" />
+      <memberOf key="att.tabDurSym.log" />
+      <memberOf key="att.tabDurSym.vis" />
+      <memberOf key="att.tabDurSym.ges" />
+      <memberOf key="att.tabDurSym.anl" />
+    </classes>
+    <content>
+      <rng:empty />
+    </content>
+    <constraintSpec ident="check_tabDurSym_in_beams" scheme="isoschematron">
+      <constraint>
+        <sch:rule context="mei:tabDurSym">
+          <sch:assert role="warning" test="not(ancestor::mei:beam)">It is very uncommon to see tab
```

Comment:
"Tab duration symbols should not occur within beams."

If it's uncommon but still allowed, then it shouldn't be a warning.

Reply:
Agreed. We have removed the warning.